MonoGame.Framework/FrameworkDispatcher.cs

```diff
@@ -2,15 +2,60 @@
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
 
+using System;
+using Microsoft.Xna.Framework.Audio;
+
 namespace Microsoft.Xna.Framework
 {
-	public static class FrameworkDispatcher
-	{
-		public static void Update()
-		{
-			// "Updates the status of various framework components
-			// (such as power state and media), and raises related events"
-		}
-	}
+    /// <summary>
+    /// Helper class for processing internal framework events.
+    /// </summary>
+    /// <remarks>
+    /// If you use <see cref="Game"/> class, <see cref="Update()"/> is called automatically.
+    /// Otherwise you must call it as part of your game loop.
+    /// </remarks>
+    public static class FrameworkDispatcher
+    {
+        private static bool _initialized = false;
+
+        /// <summary>
+        /// Processes framework events.
+        /// </summary>
+        public static void Update()
+        {
+            if (!_initialized)
+                Initialize();
+
+            DoUpdate();
+        }
+
+        private static void DoUpdate()
+        {
+            // Update sound system
+#if OPENGL
+            OpenALSoundController.GetInstance.Update();
+#endif
+            SoundEffectInstancePool.Update();

```

Comment:
That seems great, I'll update this when it gets merged. Thanks for the heads-up!